apps/designer/app/designer/features/style-panel/shared/style-info.ts

```diff
@@ -0,0 +1,37 @@
+import type { Style, StyleProperty, StyleValue } from "@webstudio-is/css-data";
+import { useMemo } from "react";
+
+export type StylePropertyInfo = {
+  value: StyleValue;
```

Comment:
this value is the `computed`, right? it seems for the consumers, it's now unclear what the value is,  because you don't see how its defined

Reply:
It's computed from browser and local style. Later it will be computed from inherited and cascaded

Reply:
computed from the browser always results in the final value, when all things come together, right?

Reply:
Yes, but we don't wanna always use it because browser computes in pixels.

Reply:
So everything we can find in data will give final result and then fallback to computed from browser

Reply:
sure, but then we shouldn't be using computed as a value at all?